pkg/controller/reaper.go

```diff
@@ -180,6 +183,10 @@ func (c *Reaper) reconcile(key string) error {
 	if !ns.ObjectMeta.DeletionTimestamp.IsZero() {
 		return nil
 	}
+	if createdByCIOperatorAndExceedMaxAge(ns, time.Now().Add(-c.maxCIOpProjectAge)) {
+		logger.Info("namespace is created by ci-operator and exceeded the max age, deleting")
+		return c.client.CoreV1().Namespaces().Delete(context.TODO(), name, metav1.DeleteOptions{})
```

Comment:
```suggestion
		//return c.client.CoreV1().Namespaces().Delete(context.TODO(), name, metav1.DeleteOptions{})
```

@bbguimaraes 

Maybe comment the deleting out and check the logs as dry-run? Need to be cautious for deleting.

Reply:
I am thinking if we could try it only on 1 cluster.

Reply:
Logging for a while before deletion is activated should be fine.

It would take some time to gather data, but the time span for this problem is months/years anyway.

Reply:
Done.